src/api/controllers/windows/ComponentController.ts

```diff
@@ -0,0 +1,129 @@
+// import { ComponentPair, ElementData } from '../../../components';
```

Comment:
Another commented out file, why?

Reply:
it is not implemented yet, and will not be in the scope of this PR, however it was causing issues with the react compiler that were preventing me from building/running the project. Commented out until implementation is completed.